src/services/symbolDisplay.ts

```diff
@@ -161,6 +163,12 @@ function getSymbolKindOfConstructorPropertyMethodAccessorFunctionOrVar(typeCheck
         else if (symbol.valueDeclaration && isVarConst(symbol.valueDeclaration as VariableDeclaration)) {
             return ScriptElementKind.constElement;
         }
+        else if (symbol.valueDeclaration && isVarUsing(symbol.valueDeclaration as VariableDeclaration)) {
+            return ScriptElementKind.variableUsingElement;
+        }
+        else if (symbol.valueDeclaration && isVarAwaitUsing(symbol.valueDeclaration as VariableDeclaration)) {
+            return ScriptElementKind.variableAwaitUsingElement;
+        }
         else if (forEach(symbol.declarations, isLet)) {

```

Comment:
Honestly, we should just make these non if/else.

Reply:
Yeah it seems a little whack to else on a block that clearly returns, but, the entire section of code does that so I'm not gonna complain 😄